packages/bun-types/bun.d.ts

```diff
@@ -3299,6 +3306,13 @@ declare module "bun" {
      */
     static readonly byteLength: 32;
   }
+  class SHA512_224 extends CryptoHashInterface<SHA512_224> {
```

Comment:
we could just not add this version, and let someone complain about the lack of it